sdk/log/record.go

```diff
@@ -97,12 +101,10 @@ type Record struct {
 }
 
 func (r *Record) addDropped(n int) {
-	logAttrDropped()
 	r.dropped += n
```

Comment:
would it not be easier to keep `logAttrDropped()` here just add it inside an `if`?

```go
func (r *Record) addDropped(n int) {
	r.dropped += n
	if n > 0 {
		logAttrDropped()
	}
```